src/components/Charts/utils.ts

```diff
@@ -0,0 +1,172 @@
+import _ from 'lodash';
+import Highcharts, { dateFormat } from 'highcharts';
+import palette from '../../assets/theme/palette';
+
+export const last = (list: any[]) => list[list.length - 1];
+
+export const formatDecimal = (num: number, places = 2): string =>
+  num.toFixed(places);
+
+export const formatPercent = (num: number): string =>
+  `${formatDecimal(100 * num, 1)} %`;
+
+export const parseDate = (date: Date): number => new Date(date).valueOf();
+
+export const currentValueAnnotation = (
+  x: number,
+  y: number,
+  text: string,
+): Highcharts.AnnotationsOptions => ({
+  labelOptions: {
+    backgroundColor: palette.white,
+  },
+  labels: [
+    {
+      align: 'left',
+      // @ts-ignore - Bug in Highchart types
+      verticalAlign: 'center',
+      x: 12,
+      shape: 'rect',
+      point: {
+        xAxis: 0,
+        yAxis: 0,
+        x,
+        y,
+      },
+      style: {
+        fontSize: '14px',
+        fontWeight: 'bold',
+      },
+      text,
+      className: 'ZoneAnnotation ZoneAnnotation--CurrentValue',
+    },
+  ],
+});
+
+export const getTickPositions = (
+  minY: number,
+  maxY: number,
+  zones: Zone[],
+): (number | undefined)[] =>
+  zones
+    .map(zone => zone.value)
+    .filter(val => !_.isUndefined(val))
+    .concat([minY, maxY])
+    .sort();
+
+export const getMaxY = (data: Highcharts.Point[]) => _.max(data.map(d => d.y));
+
+export const roundAxisLimits = (axisMin: number, axisMax: number) => [
+  axisMin,
+  _.ceil(1.2 * axisMax, 1),
+];
+
+export const baseOptions: Highcharts.Options = {
+  title: {
+    text: undefined,
+  },
+  subtitle: {
+    text: undefined,
+  },
```

Comment:
i am confused why these aren't just the title and subtitle of the chart, but that's moot

Reply:
That's a good question, I feel like having that outside the chart gives us more flexibility in terms of styling, but we can reconsider if just using the title and subtitle of the chart would be easier.